ResearchKit/Common/ORKOrderedTask.h

```diff
@@ -577,6 +578,60 @@ typedef NS_OPTIONS(NSUInteger, ORKPredefinedTaskLimbOption) {
                                            options:(ORKPredefinedTaskOption)options;
 
 /**
+ Returns a predefined task that tests the participant's reaction time and impulsiveness.
+ 
+ In a go no go task, the participant is asked to move the device sharply in any
+ direction in response to a visual cue. You can use this task to accurately assess the participant's
+ simple reaction time. Randomly they will be given a cue to not shake the device, this tests 
+ impulsiveness
+ 
+ A go no go time task finishes when the participant has completed the required
+ number of attempts successfully. For a go, an attempt is successful when the participant exerts acceleration
+ greater than `thresholdAcceleration` to the device after the stimulus has been delivered and before
+ `timeout` has elapsed. An attempt is unsuccessful if acceleration greater than
+ `thresholdAcceleration` is applied to the device before the stimulus or if this does not occur
+ before `timeout` has elapsed. If unsuccessful, the result is not reported and the participant must
+ try again to proceed with the task. For a no go, the sucess condition is reversed
+ 
+ Data collected by the task is in the form of ORKGoNoGOTimeResult objects. These
+ objects contain if it was a go or no go if it was a success, and a timestamp representing the delivery,
```

Comment:
"contain if" --> "indicate whether" ?
missing comma after "[...] or no go" if you mean it contains that information as well as whether it was a success? Otherwise maybe parens around "if it was a success" to indicate the preceding information is only included in case of success? Basically this parses ambiguously and I find it confusing.